### ISSUE TITLE
Generates pvrtc texture with all zeros

### ISSUE THREAD

**Issue by DrMoriarty** [godotengine/godot#38040]
**Godot version:**
Godot 3.2.2.beta1

**OS/device including version:**
MacOSX 10.14.6 (18G95)

**Issue description:**
Set texture VRAM compressed, enable pvrtc compression. Reimport texture, after GDST header it contains only zeros.

Original file:
![background](https://user-images.githubusercontent.com/1177068/79743861-041cd280-830e-11ea-8804-b4732dd2168c.png)

Compressed file:

[background.png-14973fd1de8b151436931643db6aa461.pvrtc.stex.zip](https://github.com/godotengine/godot/files/4502921/background.png-14973fd1de8b151436931643db6aa461.pvrtc.stex.zip)

**Steps to reproduce:**

**Minimal reproduction project:**

**Comment by DrMoriarty** [comment]
It was broken after 3.1.1
For example this file generated by godot-3.1.1.stable:
[background.png-14973fd1de8b151436931643db6aa461.pvrtc.stex.zip](https://github.com/godotengine/godot/files/4503019/background.png-14973fd1de8b151436931643db6aa461.pvrtc.stex.zip)

**Comment by DrMoriarty** [comment]
I'm trying to fix it and it seems that our pvrtc encoder completely broken.
So my game looks on an emulator:
![Снимок экрана 2020-04-20 в 23 21 06](https://user-images.githubusercontent.com/1177068/79796120-a7e19f00-835d-11ea-907b-c38662bda6c2.png)

And this is screenshot from iphone:
![Screen Shot 2020-04-20 at 23 19 12](https://user-images.githubusercontent.com/1177068/79796146-b760e800-835d-11ea-9310-a46c00772d90.png)

One texture (container in the center of screen) is vram packed.

**Comment by Zireael07** [comment]
IIRC VRAM packing isn't supported on mobiles?

**Comment by DrMoriarty** [comment]
@Zireael07 what is it IIRC?

**Comment by Zireael07** [comment]
IIRC = If I recall correctly

**Comment by DrMoriarty** [comment]
@Zireael07 therefore there is a reason to make it workable.

**Comment by Calinou** [comment]
Related to #28669?

**Comment by DrMoriarty** [comment]
@Calinou yes, exactly. Now I'm working on new pvrtc compressor for godot. Hope it will be ready in couple of days.
